python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/dbt_packages/test_columns_metadata.py

```diff
@@ -74,3 +76,29 @@ def assert_columns_metadata(context: AssetExecutionContext, dbt: DbtCliResource)
     )
 
     assert result.success
+
+
+def test_dbt_cli_no_jinja_log_info() -> None:
+    dbt = DbtCliResource(project_dir=os.fspath(test_metadata_path))
+    dbt_cli_parse_invocation = dbt.cli(["parse"])
+
+    assert dbt_cli_parse_invocation.is_successful()
+    assert not any(
+        event.raw_event["info"]["name"] == "JinjaLogInfo"
+        for event in dbt_cli_parse_invocation.stream_raw_events()
+    )
+
+
+def test_dbt_cli_no_jinja_log_info_raw() -> None:
+    # Ensure `log_columns_in_relation.sql` does not produce empty `{}` statements when run outside of the context of `dagster-dbt`
+    result = subprocess.run(
+        ["dbt", "parse", "--no-partial-parse", "--no-use-colors"],
```

Comment:
Regex seems unnecessary: let's emit the logs as JSON and parse those for `JinjaLogInfo` events.

```suggestion
        ["dbt", "--log-format", "json", "--no-partial-parse", "parse"],
```

Reply:
Gooood call.